python/ray/_private/runtime_env.py

```diff
@@ -24,6 +26,42 @@
 FILE_SIZE_LIMIT = 50 * 1024 * 1024  # 50MB
 
 
+class RuntimeEnvDict:
+    """Parses and validates runtime env dictionary from the user."""
+
+    def __init__(self, runtime_env_json: dict):
+        if "conda" in runtime_env_json:
+            self.conda = runtime_env_json["conda"]
+        else:
+            self.conda = None
+        if "working_dir" in runtime_env_json:
+            self.working_dir = runtime_env_json["working_dir"]
+        else:
+            self.working_dir = None
```

Comment:
nit: should this be clearer? `self.working_dir = runtime_env_json.get("working_dir")`

Reply:
Yeah agreed. We'll probably have to rewrite this entire thing though once we add the schema validation.